CONTRIBUTING.md

```diff
@@ -3,12 +3,12 @@
 Interested in contributing? Great! Here are some suggestions to make it a good
 experience:
 
-Start by [opening an issue](https://github.com/DavidAnson/markdownlint/issues),
-whether to identify a problem or outline a change. That issue should be used to
-discuss the situation and agree on a plan of action before writing code or
-sending a pull request. Maybe the problem isn't really a problem, or maybe there
-are more things to consider. If so, it's best to realize that before spending
-time and effort writing code that may not get used.
+Start by [opening an issue], whether to identify a problem or outline a change.
```

Comment:
I'm going to ignore edits like this per my previous comment in the issue. I don't think we should roll these changes into the new rule commit.

Reply:
```suggestion
Start by [opening an issue](https://github.com/DavidAnson/markdownlint/issues),
```